### ISSUE TITLE
semmeddb: generating x-bte annotations using metatriple file

### ISSUE THREAD

**Issue by colleenXu** [biothings/biothings_explorer#644]
@erikyao has [code](https://github.com/biothings/semmeddb/blob/main/pred_meta_triples.ipynb) to create a meta-triple table file ([original ask](https://github.com/biothings/pending.api/issues/63#issuecomment-1100469563)) during the semmeddb api deployment process. 

Yao got me the meta-triple file for the data used in semmeddb/semmeddb2 during Jan/Feb 2023. 

My task is to refactor the x-bte-annotation generating notebook to use this file.

Benefits:
* will add operations for the ncbigene namespace FINALLY
* get code that can be reused for auto-generating x-bte annotations! (but note that one needs to tweak / test query structure for other APIs)

Notes:
* will be setting an arbitrary count cutoff for creating operations from meta-triples. This cutoff will need adjusting because semmddb2 will have a diff number of records for sure

**Comment by colleenXu** [comment]
Done and deployed (registration refreshed): https://github.com/NCATS-Tangerine/translator-api-registry/commit/b08b9b3ff61ddab746bd6175e006e89c7dfbed7d

It uses a new metatriple + counts file generated by Yao Monday evening. However, it still uses predication counts (not the record / triple counts of the current API). I still use a cutoff of predication counts > 100

I still kept the original notebook (renamed to AutoGen_SEMMEDDB_Old.ipynb). 

The custom processing we still do (outside of Translator-curated exclusions) is written up in the collapsed section of:
https://github.com/biothings/biothings_explorer/issues/669#issuecomment-1669117440

----

EDIT: added comment

I think this effort added 739 operations (from [3936](https://github.com/NCATS-Tangerine/translator-api-registry/blob/0b083a9f814e36f8e5b1fa4635ec64a5df969202/semmeddb/generated_list.yaml) to [4675](https://github.com/NCATS-Tangerine/translator-api-registry/blob/b08b9b3ff61ddab746bd6175e006e89c7dfbed7d/semmeddb/generated_list.yaml)), which would all be Gene-related (have NCBIGene ID-namespace as either input, output, or both). 

It's a little hard to tell because the ordering of operations also changed when using the metatriples file.

**Comment by colleenXu** [comment]
Going to leave this open, since I'm waiting for a response from @erikyao on generating a metatriple file with:
* number of API records (basically unique triple count), rather than what seems like predication count
* number of API records where pmid_count > 3 (helpful given the [current constraint](https://github.com/NCATSTranslator/Feedback/issues/100#issuecomment-1632806388))

If this isn't possible, then this issue can be closed as completed. 
If this can be done, then maybe this issue can be kept open until that file is made, I adjust the notebook to use it, and generate / deploy x-bte annotations based on this data...

**Comment by colleenXu** [comment]
Related to #645

**Comment by colleenXu** [comment]
[Yao generated the file with the counts](https://github.com/biothings/biothings_explorer/issues/644#issuecomment-1670049086), and I saved a copy of this file to the same repo as the notebook. https://github.com/NCATS-Tangerine/translator-api-registry/commit/205e32db2c5109d1226ac620d76e00b5c3441677

I also adjusted the notebook to use this file, and with discussion from Andrew (recorded in the notebook), we set the cutoff to metatriples w/ >20 documents with pmid_count > 3. See the commit link for details. 

I've deployed the updated SmartAPI yaml (refreshed registration). 

So we can close this issue as complete. We can open new issues when we want to update the metatriples+counts file, notebook, and x-bte annotations / SmartAPI yaml...
